tests/unit/components/common/bs-form-test.js

```diff
@@ -0,0 +1,59 @@
+import { setupTest, test } from 'ember-qunit';
+import { reject, resolve } from 'rsvp';
+import { module } from 'qunit';
+
+module('Unit | Component | bs-form', function(hooks) {
+  setupTest(hooks);
+
+  test('calling form.validate returns success if validations pass', function(assert) {
+    assert.expect(1);
+
+    let Factory = this.owner.factoryFor('component:bs-form');
+    let subject = Factory.create({ hasValidator: true, model: {}, validate: function() { return resolve(); } });
```

Comment:
No need actually to wrap it in another function I think... `validate: resolve` would be enough. But I have been doing this in other places as well, so no big deal 😉